lm_eval/evaluator.py

```diff
@@ -126,6 +135,8 @@ def evaluate(lm, task_dict, provide_description=None, num_fewshot=0, limit=None,
     requests = collections.defaultdict(list)
     requests_origin = collections.defaultdict(list)
 
+    overlaps = collections.defaultdict(list) # {task_name: contaminated_docs}
```

Comment:
i.e this could be an attribute on that class

Reply:
I feel like this is fairly appropriate inside evaluate, as above please refactor if desired.